pymongo/topology.py

```diff
@@ -128,15 +145,17 @@ def __init__(self, topology_settings):
         self._opened = False
         self._closed = False
         self._lock = _create_lock()
+        assert self._settings.condition_class is not None
         self._condition = self._settings.condition_class(self._lock)
-        self._servers = {}
-        self._pid = None
-        self._max_cluster_time = None
+        self._servers: Dict[_Address, Server] = {}
+        self._pid: Optional[int] = None
+        self._max_cluster_time: Optional[Mapping[str, Any]] = None
```

Comment:
Can we add a type alias for cluster time called ClusterTime?

Reply:
done!